index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+  
+    <main id="main">
+    <div class = "title_paragraph">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+    </div>
+     
+      <figure id="img-div">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" id = "image">
+        <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+      </figure>
+      <div id="tribute-info">
+        <h2>Here's a time line of Dr. Borlaug's life:</h2>
+           <ul>
+      <li><strong>1914</strong> - Born in Cresco, Iowa</li>
+      <li>
+        <strong>1933</strong> - Leaves his family's farm to attend the
+        University of Minnesota, thanks to a Depression era program known as the
+        "National Youth Administration"
+      </li>
+      <li>
+        <strong>1935</strong> - Has to stop school and save up more money. Works
+        in the Civilian Conservation Corps, helping starving Americans. "I saw
+        how food changed them", he said. "All of this left scars on me."
```

Comment:
I think it would be better to use more semantic tags and separate the title from the main tag. inside the main, you can add sections tags on your page and use div to have more control of the element when you want to position it anywhere of the page and create a footer for the page. you can follow this link to get valuable information https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md

Reply:
Great work 👍🏿 
- [ ] Please consider the comment above about the use of polite words ⬆️